### PR TITLE
feat: add agETH token

### PR DESCRIPTION
**Pull request by galacticminter**
Adding Kelp's agETH config to whitelist on ITS.

### CODE REVIEW

**Review by Olanetsoft — CHANGES_REQUESTED** [review]
@galacticminter, kindly add your logo/icon.

**Comment by galacticminter** [comment]
Hi @Olanetsoft, am I able to call these tokens on L2s as axl_agETH ? Is that convention fine ?

**Comment by Olanetsoft** [comment]
> Hi @Olanetsoft, am I able to call these tokens on L2s as axl_agETH ? Is that convention fine ?

Any reason for doing this?

**Comment by galacticminter** [comment]
We have a canonical token called agETH or wagETH on these L2s. Wouldn't want axelar tokens to be called with the same name.

**Comment by Olanetsoft** [comment]
> We have a canonical token called agETH or wagETH on these L2s. Wouldn't want axelar tokens to be called with the same name. @Olanetsoft

@galacticminter I see. You can't rename the token `agETH` on a chain, i see that's what its registered as [here](https://etherscan.io/token/0xe1b4d34e8754600962cd944b535180bd758e6c2e). You won't be able to change the chain name on the chain itself. ERC20s don't support something like a changeName function. 

Alternatively, if you have a UI, you can map the proposed name to the name on-chain, e.g., `"agETH": "axl_agETH," so that any time someone tries to load agETH, you can show on your UI the axl-prefixed name.

**Comment by galacticminter** [comment]
Sounds good @Olanetsoft 

Could you please approve & merge this?

**Review by Olanetsoft — APPROVED** [review]

**Comment by Olanetsoft** [comment]
@galacticminter, one last thing. Could you please add your coingecko id ? eg. "coinGeckoId": "your-id".

**Comment by ecdsafu** [comment]
@galacticminter on the Squid front end they'll be displayed using a prettySymbol, that we can edit as you like. Please let me know the preferred symbol on each chain

**Review by ecdsafu — APPROVED** [review]
lgtm

**Comment by galacticminter** [comment]
Please use the pretty symbol as axl_agETH

**Comment by ecdsafu** [comment]
> Please use the pretty symbol as axl_agETH

Sounds good. In Squid we are switching to the standard agETH.axl is that ok with you? We think it reads much better and is consistent with other bridge notations, such as USDC.e for ESDU bridged from ethereum native bridge

**Comment by canhtrinh** [comment]
> > Please use the pretty symbol as axl_agETH
> 
> Sounds good. In Squid we are switching to the standard agETH.axl is that ok with you? We think it reads much better and is consistent with other bridge notations, such as USDC.e for ESDU bridged from ethereum native bridge

@ecdsafu - Jake from our BD team confirmed with theirs that your proposal is okay. Do you need to reflect that symbol here or do you have a separate mapping elsewhere that does that? If the latter, I will merge this PR rn

**Comment by Olanetsoft** [comment]
> > > Please use the pretty symbol as axl_agETH
> > 
> > 
> > Sounds good. In Squid we are switching to the standard agETH.axl is that ok with you? We think it reads much better and is consistent with other bridge notations, such as USDC.e for ESDU bridged from ethereum native bridge
> 
> @ecdsafu - Jake from our BD team confirmed with theirs that your proposal is okay. Do you need to reflect that symbol here or do you have a separate mapping elsewhere that does that? If the latter, I will merge this PR rn

Friendly bump @ecdsafu ^

**Review by ecdsafu — COMMENTED** [review]
Please update the prettySymbol to be agETH.axl and then we can merge and release

File: registry/mainnet/interchain/squid.tokenlist.json [hunk 1]
```diff
@@ -2549,6 +2549,86 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x1ac831937426660d5952bc9ae0dc7c895a3c3f5f159d3518813df30181404be6": {
+      "tokenId": "0x1ac831937426660d5952bc9ae0dc7c895a3c3f5f159d3518813df30181404be6",
+      "deployer": "0x7AAd74b7f0d60D5867B59dbD377a71783425af47",
+      "originalMinter": null,
+      "prettySymbol": "agETH",
```
Comment (canhtrinh) [inline registry/mainnet/interchain/squid.tokenlist.json:2557]:
```suggestion
      "prettySymbol": "agETH.axl",
```

**Review by canhtrinh — APPROVED** [review]

**Pull request merged**